distributed/dashboard/tests/test_scheduler_bokeh.py

```diff
@@ -610,6 +611,75 @@ async def test_TaskGraph_order(c, s, a, b):
     assert gp.node_source.data["state"][gp.layout.index[y.key]] == "erred"
 
 
+@gen_cluster(client=True)
+async def test_TGroupGraph(c, s, a, b):
+    tgg = TGroupGraph(s)
+    futures = c.map(inc, range(10))
+    await wait(futures)
+
+    tgg.update()
+    dnode = dict(tgg.nodes_source.data)
```

Comment:
`tgg.nodes_source.data` is `dict`-like, so there's no need to explicitly convert it to a `dict` here (and in other places). Instead we can just used `tgg.nodes_source.data` directly. 

```diff
diff --git a/distributed/dashboard/tests/test_scheduler_bokeh.py b/distributed/dashboard/tests/test_scheduler_bokeh.py
index be161ea8..0dc916f9 100644
--- a/distributed/dashboard/tests/test_scheduler_bokeh.py
+++ b/distributed/dashboard/tests/test_scheduler_bokeh.py
@@ -618,22 +618,19 @@ async def test_TGroupGraph(c, s, a, b):
     await wait(futures)

     tgg.update()
-    dnode = dict(tgg.nodes_source.data)
-    assert all(len(L) == 1 for L in dnode.values())
-    assert dnode["name"] == ["inc"]
-    assert dnode["tot_tasks"] == [10]
+    assert all(len(L) == 1 for L in tgg.nodes_source.data.values())
+    assert tgg.nodes_source.data["name"] == ["inc"]
+    assert tgg.nodes_source.data["tot_tasks"] == [10]

-    darr = dict(tgg.arrows_source.data)
-    assert all(len(L) == 0 for L in darr.values())
+    assert all(len(L) == 0 for L in tgg.arrows_source.data.values())

     futures2 = c.map(dec, range(5))
     await wait(futures2)

     tgg.update()
-    dnode = dict(tgg.nodes_source.data)
-    assert all(len(L) == 2 for L in dnode.values())
-    assert dnode["name"] == ["inc", "dec"]
-    assert dnode["tot_tasks"] == [10, 5]
+    assert all(len(L) == 2 for L in tgg.nodes_source.data.values())
+    assert tgg.nodes_source.data["name"] == ["inc", "dec"]
+    assert tgg.nodes_source.data["tot_tasks"] == [10, 5]

     del futures, futures2
     while s.task_groups:
```